draft-ietf-tls-tls13.md

```diff
@@ -2301,14 +2303,22 @@ EdDSA algorithms
 : Indicates a signature algorithm using EdDSA as defined in
   {{RFC8032}} or its successors. Note that these correspond to the
   "PureEdDSA" algorithms and not the "prehash" variants.
+
+Legacy algorithms
+: Indicates algorithms which are being deprecated because they use
+  SHA-1 but are still permitted for backward compatibility,
+  specifically RSA using RSASSA-PKCS1-v1_5 and ECDSA.  These values
```

Comment:
I don't think this statement as written is accurate.
You can use either RSA or ECDSA without SHA-1, which is allowed in the real list of defined algorithms.